Makefile

```diff
@@ -0,0 +1,9 @@
+restore:
```

Comment:
I believe there is some `godep update`. We can ask madhu how he doing this in his projects.